sources/org.osbuild.files

```diff
@@ -30,12 +30,14 @@ def fetch(url, checksum, directory):
     if os.path.isfile(f"{directory}/{checksum}"):
         return
 
+    secrets = url.get("secrets")
+    url_path = url.get("url")
```

Comment:
Should probably rename `url` into something more descriptive, because it is not a plain URL anymore.

Reply:
Will use `target` instead of `urls`

Reply:
I am keeping `url`